### PR TITLE
Add new bio for glennon

### PR DESCRIPTION
**Pull request by glennon**

### CODE REVIEW

File: bios/glennon-bio.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+Alan Glennon
+
+- A bio about me written by others: [bio](https://en.wikipedia.org/wiki/John_Alan_Glennon)
```
Comment (rfriedel) [inline bios/glennon-bio.md:3]: Nice bio link .. I wish I would be mentioned in wikipedia :+1: 

Very interesting work you do!

**Comment by glennon** [comment]
:+1:  self checks passed

**Comment by rfriedel** [comment]
Ha ha .. you didn't change the file .. just commented ;)

**Comment by rfriedel** [comment]
gotta click on that little pen once you selected the file :clap:

**Comment by glennon** [comment]
The edits were tiny. Full last name to last name initial.

**Pull request merged**